calico/felix/felix.py

```diff
@@ -411,9 +393,7 @@ def handle_endpointdestroyed(self, message, sock):
             return
 
         # Unsubscribe endpoint.
-        sock = self.sockets[Socket.TYPE_ACL_SUB]
-        sock._zmq.setsockopt(zmq.UNSUBSCRIBE, delete_id.encode('utf-8'))
-
+        self.sockets[Socket.TYPE_ACL_SUB].unsubscribe(delete_id.encode('utf-8'))
```

Comment:
It is OK to get ACL updates when the endpoint does not exist; I also suspect it may be possible to get a destroy for an endpoint which does not exist. We might (for example) get the destroy but then find that it arrives after the completion of a resync which gets rid of the same endpoint. Or we might have lost the creation because of an earlier issue for which a resync has been initiated.

I therefore think we should just log it as surprising and continue.